README.md

```diff
@@ -78,26 +78,29 @@ To use GraphScope, we need to establish a session in a python interpreter.
 import os
 import graphscope
 
-# Setting an env for mounting test data from local disk,
-# hence we can access data inside the pods.
-
 # assume we mount `~/test_data` to `/testingdata` in pods.
 k8s_volumes = {
     "data": {
         "type": "hostPath",
         "field": {
-          "path": os.path.expanduser("~/test_data/"),
-          "type": "Directory"
+            "path": os.path.expanduser("~/test_data/"),
+            "type": "Directory"
         },
         "mounts": {
-          "mountPath": "/testingdata"
+            "mountPath": "/testingdata"
         }
     }
 }
 
 sess = graphscope.session(k8s_volumes=k8s_volumes)
 ```
 
+for macOS, session need to established with LoadBalancer service type (default is NodePort).

```

Comment:
For macOS, the session needs to establish with the `LoadBalancer` service type (which is `NodePort` by default).